### PR TITLE
Added comparator for geometry edits to motorways and trunk roads

### PR DESCRIPTION
**Pull request by manoharuss**
@amishas157 can you take a quick look once to correct and optimize. I added a compare function and a single fixture for this. Will add more fixtures and update here in this pr.

cc @maning @chtnha @planemad

### CODE REVIEW

File: comparators/motorway_trunk_geometry_changed.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+function geometryChanged(newVersion, oldVersion) {
+  if (!oldVersion && newVersion.deleted) {
+    return false;
+  }
+
+  if (!oldVersion && !newVersion.deleted) {
+    return false;
+  }
```
Comment (amishas157) [inline comparators/motorway_trunk_geometry_changed.js]: 👋 , can you explain when you would like to run this compare function, feature addition/modification/deletion ?

File: comparators/motorway_trunk_geometry_changed.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  if (isMajorRoad(oldVersion)) {
+    if (!deepEquals(getGeometry(oldVersion), getGeometry(newVersion))) {
```
Comment (amishas157) [inline comparators/motorway_trunk_geometry_changed.js]: Nice! 👍

File: comparators/motorway_trunk_geometry_changed.js (hunk 1 above)
Comment (manoharuss) [reply comparators/motorway_trunk_geometry_changed.js]: only for geometry changes @amishas157

Comment (manoharuss) [reply comparators/motorway_trunk_geometry_changed.js]: Yeah that line read odd for me as well @amishas157 going to change it

**Comment by amishas157** [comment]
Thanks @manoharuss for working on this 🎉

**Pull request merged**